Work/pcost.py

```diff
@@ -1,3 +1,31 @@
 # pcost.py
 #
-# Exercise 1.27
+# Exercise 1.27 + 1.30 + 1.31 + 1.32
+import sys
+def portfolio_cost(filename):
+    import csv
+    counterOfLines = 0
+    Total_cost = 0.00
+    try:
+        with open(filename) as f:
+            rows = csv.reader(f)
+            headers = next(rows)
+            for row in rows:
```

Comment:
у тебя до этого было хорошо сделано, когда ты три пременных из одной строки инитил, тут тоже можно так:

```suggestion
            for name, shares, price in rows:
```

Reply:
ну и дальше соответственно поправить

Reply:
@obopodo хорошо, спасибо. Сфокусировался на задаче, а там было просто переделать из одного в другое.